src/query/settings/src/settings_default.rs

```diff
@@ -155,6 +155,12 @@ impl DefaultSettings {
                     possible_values: None,
                     display_in_show_settings: true,
                 }),
+                ("inlist_to_subquery", DefaultSettingValue {
```

Comment:
No need this setting and check the IN list length to enable or disable?

Reply:
yes, i need some time to find a proper threshold, then remove the setting